src/Middleware/Spa/SpaProxy/src/SpaProxyLaunchManager.cs

```diff
@@ -74,8 +74,8 @@ public async Task<bool> IsSpaProxyRunning(CancellationToken cancellationToken)
     {
         var httpClient = CreateHttpClient();
 
-        using var timeout = new CancellationTokenSource(TimeSpan.FromSeconds(10));
-        using var cancellationTokenSource = CancellationTokenSource.CreateLinkedTokenSource(timeout.Token, cancellationToken);
+        using var cancellationTokenSource = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken);
+        cancellationTokenSource.CancelAfter(TimeSpan.FromSeconds(10));
```

Comment:
We keep doing this, it seems like we want a new overload for this😆. @stephentoub 
`CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, TimeSpan.FromSeconds(10));`